Dockerfile

```diff
@@ -62,5 +62,6 @@ COPY --from=builder /build/templates /templates/
 COPY --from=builder /build/build/release/pipeline /
 COPY --from=builder /build/build/release/worker /
 COPY --from=builder /build/build/release/pipelinectl /
+COPY config/anchore/policies/ /policies/
```

Comment:
**Note**: this is fine for now, even consistent with the `templates` directory, but in the long run it might clash with other non-anchore policies as `policy` is a widely used term, so qualification might be required in the future.

Reply:
You are right. There will be a quite big refactor in the Anchore user management and it will have an effect on the default policy handling as well. So we will change this at that point.